openvm/src/lib.rs

```diff
@@ -486,19 +487,20 @@ pub enum ExtendedVmConfigPeriphery<F: PrimeField32> {
 
 #[derive(Clone, Serialize, Deserialize, Default, Debug, Eq, PartialEq)]
 pub struct AirMetrics {
+    /// The main AIR statistics (i.e., number of main columns, constraints, and bus interactions)
+    pub air_stats: AirStats,
+    /// The width of the AIR, including OpenVM specific columns
+    /// (i.e., preprocessed and log_up, in addition to the main columns)
```

Comment:
This is a bit redundant... Probably it would be better to:
- Remove preprocessed columns, because I think they should always be 0, and even if they are not, we probably don't want to count them?
- Store the number of all columns in `AirStats` (main + logup, if they exist).

VMs with LogUp (no GKR) (e.g. OpenVM) would then be able to use the number of columns as a good proxy for the cost, VMs with LogUp + GKR would have to specify the cost per bus interaction separately.

But this seems like a bigger change.